Extensions/TileMap/JsExtension.js

```diff
@@ -45,6 +45,8 @@ module.exports = {
       if (propertyName in objectContent) {
         if (typeof objectContent[propertyName] === 'boolean')
           objectContent[propertyName] = newValue === '1';
+        else if (typeof objectContent[propertyName] === 'number')
```

Comment:
The issue with this is that if something/somebody make a mistake in the code and store the property to a string, then it will never be changed back to a number? It will stay, wrongly, a string forever. This includes games that have saved "layerIndex" as a string.

This is why I recommend going back to an explicit approach as I mentioned in the other thread.

Reply:
i updated that already :)